comictaggerlib/taggerwindow.py

```diff
@@ -940,6 +940,9 @@ def form_to_metadata(self) -> None:
             md.add_credit(name, role, bool(primary_flag))
 
         md.pages = self.page_list_editor.get_page_list()
+
+        # Preserve hidden md values
+        md.overlay(self.metadata, mode=comicapi.merge.Mode.ADD_MISSING, merge_lists=False)
```

Comment:
This doesn't account for when someone wants to remove something, how would I remove the title from a comic that already has one set